EduPerfTests/Program.cs

```diff
@@ -158,13 +230,17 @@ static void DeterminePerformanceIterations()
             if (performanceIterations == 0) DeterminePerformanceIterations();
         }
 
-        static void DeterminePageLoadSites()
+        static void DetermineSitesToLoad()
         {
-            Console.WriteLine("Specify the path to the csv file containing the sites you would like to test (blank to skip):");
-            var path = Console.ReadLine();
+            Console.WriteLine("Specify the path to the csv file containing the sites you would like to test (blank to skip, d for default):");
+            pathToSites = Console.ReadLine();
+
+            if (string.IsNullOrWhiteSpace(pathToSites)) return;
 
-            if (string.IsNullOrWhiteSpace(path)) return;
-            using (var reader = new StreamReader(path))
+            if (pathToSites.ToLower() == "d")
+                pathToSites = @"c:\users\johnjan\desktop\sitelist.csv";
```

Comment:
totally agree. I didn't mean to check in the change pointing to my drive :-). I think we should default to current directory. I'll fix.